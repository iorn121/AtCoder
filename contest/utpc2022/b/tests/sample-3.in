18
2092413 29557322
-83061793 -86609930
41750783 34587912
94366440 62086679
42714686 -18841496
10264522 -60895144
94721140 -72749181
-68416594 -56662164
85327102 82520146
-97103434 30343456
54136952 -27352836
-38212546 92787647
-72073444 28721824
32088712 -55984481
-38640098 22126296
84969832 -72730101
-4649799 14505075
92214627 89508662
