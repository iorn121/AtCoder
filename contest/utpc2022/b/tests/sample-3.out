225535157.051134686276782
